tutorials/shading/shading_reference/canvas_item_shader.rst

```diff
@@ -35,6 +35,17 @@ Render modes
 | **skip_vertex_transform**       | VERTEX/NORMAL/etc need to be transformed manually in vertex function.|
 +---------------------------------+----------------------------------------------------------------------+
 
+Global built-ins
+^^^^^^^^^^^^^^^^
+
+Global built-ins available everywhere, including custom functions.
```

Comment:
Should be "Global built-ins are available..." Same goes for the other instances.

Reply:
Thanks, fixed.